test/pytest/service-desktop/test_templates.py

```diff
@@ -117,6 +116,37 @@ def test_changing_template_body(self):
         assert total_count == initial_count
         assert test_passed == True
 
+    def test_changing_template_order(self):
```

Comment:
Please provide documentation for this test: what is the scenario, the needed parameters for testing and a short description. Also please rework this test to not use `assert` in the middle of the test. If there is such a need please divide this test into smaller pieces or create a parametrization for this.